components/Common/Tabs/index.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+
+import Tabs from './index';
+
+type Story = StoryObj<typeof Tabs>;
+type Meta = MetaObj<typeof Tabs>;
+
+export const Default: Story = {};
+
+export default {
+  component: Tabs,
+  args: {
```

Comment:
This args should not be here. They should be on `export const Default: Story = {};`